weekly/2016/0511.md

```diff
@@ -0,0 +1,196 @@
+
+- [EncloseJS](http://enclosejs.com/): Compile your Node.js project. No sources aboard.
+
+- [Slate](https://github.com/tripit/slate/): Beautiful static documentation for your API.
+
+- [Git Annex](https://github.com/joeyh/git-annex) and [homepage](http://git-annex.branchable.com/): Manage large files with git.
+
+# Frontend and Backend
+
+- [elm-lang](http://elm-lang.org/) and [tutorial](https://github.com/evancz/elm-architecture-tutorial): 
+the best of functional programming in your browser.
+
+- [Redux](http://redux.js.org/index.html): A predictable state container for JavaScript apps.
+
+- [material-ui](http://www.material-ui.com/): React components that implement Google's Material Design.
+
+- [JsSIP](https://github.com/versatica/JsSIP): The JavaScript SIP library.
+
+---
+
+- [resque-loner](https://github.com/resque/resque-loner): Support for unique jobs in Resque: Only one job with the same payload per queue.
+
+- [PHP Performance Crash Course, Part 1: The Basics](https://dzone.com/articles/php-performance-crash-course) and [PHP Performance Crash Course, Part 2: The Deep Dive](https://dzone.com/articles/php-performance-crash-course-0)
+
+- [linkerd](https://github.com/BuoyantIO/linkerd): A dynamic linker for microservices.
+
+- [vert.x](https://github.com/eclipse/vert.x): Vert.x is a tool-kit for building reactive applications on the JVM.
+
+- [telegram-bot-ruby](https://github.com/atipugin/telegram-bot-ruby): Ruby wrapper for Telegram's Bot API.
+
+- [Falcor](http://netflix.github.io/falcor/documentation/jsongraph.html) and [GraphQL](http://facebook.github.io/graphql/): 
+
+	>	不同于 Restful 的 Api 架构，只暴露一个接口，通过 请求的 Model 获取数据 -- [@swaylq][sway]
+
+- [WindPT](https://github.com/labs7in0/WindPT): A plugin for PHPWind 9 to build a Private BT tracker.
+
+- [Mycat-server](https://github.com/MyCATApache/Mycat-Server): An Open-Source software, “a large database cluster” oriented to enterprises.
+
+	>	一个基于阿里 [cobar](https://github.com/alibaba/cobar) 的开源 mysql 中间件 -- [@at15][at15]
+
+# Pretty Service
+
+- [Gogs](https://gogs.io/): Emphasized by [@ComMouse][dou]
+
+- [R](http://dev.af83.com/2012/07/31/should-we-namespace-redis.html)[e](http://dev.af83.com/2012/08/02/namespace-redis-with-ease.html)[d](https://github.com/soveran/nest)[i](https://github.com/resque/redis-namespace)s: 一个namespace引发的惨案
+
+- [Pusher](https://pusher.com/): We spend our time maintaining reliable and scalable realtime infrastructure so you can spend yours building awesome realtime features.
+
+	>	travis-CI使用的推送日志的工具 -- [@gaocegege][cece]
+	>	github 好像也用它 -- [@at15][at15]
+
+- [elastic-graph](https://www.elastic.co/products/graph): Explore connections in Elasticsearch data.
+
+- [VJ4](https://github.com/vijos/vj4): Next generation Vijos. Built with asyncio on Python 3.5.
+
+
+- [Telegram 中文群组列表机器人](https://github.com/jqs7/Jqs7Bot)
+
+	>	Telegram 群里一直出没的 "TG群组娘". --[@at15][at15]
```

Comment:
It was me... https://github.com/dyweb/web-stuff/issues/68#issuecomment-217784362